File: examples/sms.cs
using System;
using System.Net.Http;
using System.Threading.Tasks;

class Program
{
    private static readonly HttpClient client = new HttpClient();

    static async Task Main()
    {
        string apiKey = "YOUR_API_KEY";  // Replace with your actual API key
        string mobile = Prompt("Enter the mobile number (format: 01XXXXXXXXX): ");
        string message = Prompt("Enter the message to send: ");

        await SendSms(apiKey, mobile, message);
    }

    static async Task SendSms(string apiKey, string mobile, string message)
    {
        string url = $"https://sms.anbuinfosec.xyz/api/sms?apikey={apiKey}&mobile={mobile}&msg={Uri.EscapeDataString(message)}";

        HttpResponseMessage response = await client.GetAsync(url);
        string result = await response.Content.ReadAsStringAsync();
        
        dynamic responseData = Newtonsoft.Json.JsonConvert.DeserializeObject(result);
        
        if (responseData.success == true)
        {
            Console.WriteLine($"Success: {responseData.message}");
            Console.WriteLine($"SMS sent to: {responseData.mobile}");
            Console.WriteLine($"Message: {responseData.msg}");
            Console.WriteLine($"New Balance: {responseData.newBalance}");
        }
        else
        {
            Console.WriteLine($"Error: {responseData.message}");
        }
    }

    static string Prompt(string message)
    {
        Console.Write(message);
        return Console.ReadLine();
    }
}

File: examples/sms.php
<?php

function sendSms($apiKey, $mobile, $message) {
    $url = "https://sms.anbuinfosec.xyz/api/sms?apikey=$apiKey&mobile=$mobile&msg=$message";

    $response = file_get_contents($url);
    $responseData = json_decode($response, true);

    if ($responseData['success']) {
        echo "Success: " . $responseData['message'] . PHP_EOL;
        echo "SMS sent to: " . $responseData['mobile'] . PHP_EOL;
        echo "Message: " . $responseData['msg'] . PHP_EOL;
        echo "New Balance: " . $responseData['newBalance'] . PHP_EOL;
    } else {
        echo "Error: " . $responseData['message'] . PHP_EOL;
    }
}

$apiKey = "YOUR_API_KEY";  // Replace with your actual API key
$mobile = readline("Enter the mobile number (format: 01XXXXXXXXX): ");
$message = readline("Enter the message to send: ");

sendSms($apiKey, $mobile, $message);
